subgraph/src/fleek-nfa.ts

```diff
@@ -25,7 +25,7 @@ import {
   NewAccessPoint as NewAccessPointEvent,
   ChangeAccessPointNameVerify as ChangeAccessPointNameVerifyEvent,
   ChangeAccessPointContentVerify as ChangeAccessPointContentVerifyEvent,
-  TokenRolesCleared as TokenRolesClearedEvent
+  TokenRolesCleared as TokenRolesClearedEvent,
```

Comment:
check this, I have the same problem on https://github.com/fleekxyz/non-fungible-apps/pull/173